ws-server/src/server.rs

```diff
@@ -51,34 +50,14 @@ impl Server {
 	pub async fn new(addr: impl ToSocketAddrs) -> Result<Self, Error> {
 		let listener = TcpListener::bind(addr).await?;
 
-		Ok(Server { listener, root: RpcModule::new() })
-	}
-
-	/// Register a new RPC method, which responds with a given callback.
-	pub fn register_method<R, F>(&mut self, method_name: &'static str, callback: F) -> Result<(), Error>
-	where
-		R: Serialize,
-		F: Fn(RpcParams) -> Result<R, CallError> + Send + Sync + 'static,
-	{
-		self.root.register_method(method_name, callback)
-	}
-
-	/// Register a new RPC subscription, with subscribe and unsubscribe methods.
-	pub fn register_subscription<F>(
```

Comment:
same question here; if we could keep this for now but create a  `temp RpcModule` internally here and append it the methods?